gsdk/src/metadata/generated.rs

```diff
@@ -2600,7 +2613,7 @@ pub mod runtime_types {
                     #[doc = "- `session_id`: id of the resume session."]
                     #[doc = "- `block_count`: the specified period of rent."]
                     resume_session_commit {
-                        session_id: ::core::primitive::u128,
+                        session_id: ::core::primitive::u32,
```

Comment:
Isn't it compatibility breakage?

Reply:
Yes, it is, however only for client code. We don't actually provide backward compatibility for gclient/gsdk. If someone needs to work with the previous runtime one will have to use the corresponding revision of the component.